prac_05/emails.py

```diff
@@ -0,0 +1,26 @@
+
+EMAIL_ADDRESS_TO_NAME = {"shibin.abraham@gmail.com": "Shibin Abraham", "digitallearning@gmail.com": "Andrew",
```

Comment:
If this is a CONSTANT, then you should not change it. We wanted a normal variable for this one.

Reply:
Thanks much.